worlds/shorthike/__init__.py

```diff
@@ -65,11 +69,19 @@ def create_items(self) -> None:
             if feather_count < 12:
                 feather_count = 12
 
-        junk = 45 - self.options.silver_feathers - feather_count - self.options.buckets
-        self.multiworld.itempool += [self.create_item(self.get_filler_item_name()) for _ in range(junk)]
         self.multiworld.itempool += [self.create_item("Golden Feather") for _ in range(feather_count)]
         self.multiworld.itempool += [self.create_item("Silver Feather") for _ in range(self.options.silver_feathers)]
         self.multiworld.itempool += [self.create_item("Bucket") for _ in range(self.options.buckets)]
+        self.multiworld.itempool += [self.create_item("Stick") for _ in range(self.options.sticks)]
+        self.multiworld.itempool += [self.create_item("Toy Shovel") for _ in range(self.options.toy_shovels)]
+
+        if self.options.random_walkie_talkie:
+            self.multiworld.itempool.append(self.create_item("Walkie Talkie"))
+        else:
+            self.multiworld.get_location("Lose Race Gift", self.player).place_locked_item(self.create_item("Walkie Talkie"))
+
+        junk = len(self.multiworld.get_unfilled_locations(self.player)) - len(self.multiworld.itempool)
```

Comment:
This does not work, discussed in the server channel in more detail

Reply:
Thanks for the heads up, this has been taken care of.